pandas/src/parser/tokenizer.c

```diff
@@ -1774,7 +1774,9 @@ double precise_xstrtod(const char *str, char **endptr, char decimal, char sci,
 double round_trip(const char *p, char **q, char decimal, char sci, char tsep,
                   int skip_trailing) {
 #if PY_VERSION_HEX >= 0x02070000
-    return PyOS_string_to_double(p, q, 0);
+    double r = PyOS_string_to_double(p, q, 0);
+    PyErr_Clear();
```

Comment:
It does.  An exception would otherwise occur in `test_float_precision_round_trip_with_text`.